Makefile

```diff
@@ -156,11 +152,15 @@ build/compiler-rt.BUILT: build/llvm.BUILT
 	DESTDIR=$(DESTDIR) ninja $(NINJA_FLAGS) -C build/compiler-rt install
 	# Install clang-provided headers.
 	cp -R $(ROOT_DIR)/build/llvm/lib/clang $(BUILD_PREFIX)/lib/
+	cp -R $(BUILD_PREFIX)/lib/clang/17/lib/wasi $(BUILD_PREFIX)/lib/clang/17/lib/wasip1
+	cp -R $(BUILD_PREFIX)/lib/clang/17/lib/wasi $(BUILD_PREFIX)/lib/clang/17/lib/wasip2
```

Comment:
Can these use `$(CLANG_VERSION)` instead of hard-coding `17`?

Reply:
Ah yes indeed! I didn't look hard enough to find that.